setup.cfg

```diff
@@ -1,4 +1,5 @@
 [pycodestyle]
+ignore = E401, W602
```

Comment:
Instead of ignoring these, can we omit the offending (presumably functional test) files from being checked?